style.css

```diff
@@ -0,0 +1,726 @@
+ font-size: 16px;
+ line-height: 16px;
+ letter-spacing: 0.07em;
+ opacity: 0.5;
+ font-family: Montserrat, sans-serif;
+ margin-bottom: 55px;
+}
+
+.clutch__description-opacity {
+ opacity: 0.5;
+ margin-bottom: 55px;
+}
+
+.box__color-cost {
+ display: flex;
+ justify-content: space-between;
+ align-items: center;
+}
+
+.circle {
+ display: inline-block;
+ margin-right: 12px;
+ border-radius: 50%;
+ height: 30px;
+ width: 30px;
+}
+
+.circle_purple {
+ background: #dedede;
+}
+
+.circle_pink {
+ background-color: #f29f9f;
+ height: 38px;
+ width: 38px;
+}
+
+.circle_blue {
+ background-color: #a8bff9;
+}
+
+.circle_green {
+ background-color: #90bd8f;
+ margin-right: 0;
+}
+
+.cost {
+ font-size: 40px;
+ line-height: 56px;
+ text-align: right;
+ font-family: Montserrat, sans-serif;
+}
+
+.btn {
+ border: 1.5px solid #000;
+ text-decoration: none;
+ font-family: Montserrat, sans-serif;
+ font-size: 13px;
+ padding: 14px 35px;
+ color: #000;
+ text-transform: uppercase;
+ display: inline-block;
+}
+
+.btn:hover {
+ box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.2);
+}
+
+.btn_cost {
+ margin-right: 15px;
+}
+
+.btn_next {
+ margin-right: 0;
+ margin-top: 40px;
+}
+
+.clutch__other {
+ background: url(img/long_arrow.png) right no-repeat;
+ text-decoration: none;
+ width: 480px;
+ font-size: 13px;
+ line-height: 90px;
+ margin-top: 50px;
+ margin-left: -50px;
+ color: #000;
+ text-transform: uppercase;
+ font-family: Montserrat, sans-serif;
+ position: absolute;
+
+}
+
+.сlutch__other: hover {
+ color: #696969;
+ cursor: pointer;
+ text-shadow: 1px 1px 2px #dcdcdc;
+}
+
+@media (max-width: 1300px) {
+ .wrap {
+   padding-right: 30px;
+   padding-left: 0;
+ }
+
+ .btn_cost {
+   display: none;
+ }
+
+ .cost {
+   font-size: 30px;
+   line-height: 46px;
+ }
+
+ .box {
+   padding: 16px 37px 25px 31px;
+   min-width: 270px;
+ }
+
+ .clutch__other {
+   display: none;
+ }
+}
+
+@media (max-width: 1100px) {
+ .bag {
+   padding-left: 0;
+   background: none;
+   width: 100%;
+ }
+
+ .box {
+   margin-left: 0;
+ }
+
+ .circle {
+   margin-right: 9px;
+   height: 23px;
+   width: 23px;
+ }
+
+ .circle_pink {
+   height: 30px;
+   width: 30px;
+ }
+}
+
+@media (max-width: 900px) {
+ .row {
+   top: 90px;
+ }
+}
+
+@media (max-width: 780px) {
+ .row {
+   flex-direction: column;
+   top: 0;
+ }
+
+ .wrap {
+   padding: 0;
+ }
+
+ .imgs {
+   width: 224px;
+ }
+
+ .wrap_part {
+   padding: 0;
+ }
+
+ .bag {
+   width: 100%;
+ }
+
+ .btn {
+   margin-left: 50px;
+ }
+
+ .btn_next {
+   display: inline-block;
+   width: 169px;
+   text-align: center;
+   margin-top: 15px;
+   margin-bottom: 0;
+   padding: 14px 50px;
+ }
+}
+
+@media (max-width: 515px) {
+ .page2 {
+   height: 596px;
+   min-width: 375px;
+ }
+
+ .imgs {
+   width: 224px;
+   height: 240px;
+   margin-bottom: 30px;
+   background: url(img/pinkbag-mobile.png) center/cover no-repeat;
+ }
+
+ .bag {
+   opacity: 0;
+ }
+
+ .box {
+   max-width: 252px;
+ }
+
+ .clutch__title {
+   font-size: 20px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)